packages/opencensus-core/src/stats/view.ts

```diff
@@ -187,9 +196,58 @@ export class BaseView implements View {
     }
   }
 
+  /**
+   * Gets view`s metric
+   * @returns {Metric}
+   */
+  getMetric(): Metric {
+    const {type} = this.metricDescriptor;
+    let startTimestamp: Timestamp;
+
+    switch (type) {
+      case MetricDescriptorType.GAUGE_INT64:
+      case MetricDescriptorType.GAUGE_DOUBLE:
+        startTimestamp = null;
+        break;
+      default:
+        const [seconds, nanos] = process.hrtime();
+        startTimestamp = {seconds, nanos};
+    }
+
+    const timeseries: TimeSeries[] = [];
+
+    Object.keys(this.rows).forEach(key => {
+      const {tags} = this.rows[key];
+      const labelValues: LabelValue[] = MetricUtils.tagsToLabelValues(tags);
+      const point: Point = this.toPoint(startTimestamp, this.getSnapshot(tags));
+      timeseries.push({startTimestamp, labelValues, points: [point]});
+    });
+
+    return {descriptor: this.metricDescriptor, timeseries};
+  }
+
+  /**
+   * Converts snapshot to point
+   * @param timestamp The timestamp
+   * @param data The aggregated data
+   * @returns {Point}
+   */
+  private toPoint(timestamp: Timestamp, data: AggregationData): Point {
```

Comment:
I would add this method on ```AggregationMetadata``` interface, so that all the AggregationData(SumData, CountData etc.) implements this method. WDYT?

Reply:
> I would add this method on `AggregationMetadata` interface, so that all the AggregationData(SumData, CountData etc.) implements this method. WDYT?

@mayurkale22 we discussed to use interfaces instead of classes as base requirement

Reply:
@isaikevych we have discussed interfaces for metric packages. I am not referring to that here, my suggestion is we need ```toPoint()``` method for all aggregationdata. It is better to add in base interface and implement in all  aggregationdata. This is more cleaner and easy approach.

Reply:
@mayurkale22 interfaces for metric packages already generated from proto by protobuf library by Issue [#133](https://github.com/census-instrumentation/opencensus-node/issues/133), PR [#188](https://github.com/census-instrumentation/opencensus-node/pull/188)
`toPoint` works works for all aggregationdata and as we discussed and marked in task that converter will work as well.